pkg/providers/amifamily/custom.go

```diff
@@ -38,8 +39,8 @@ func (c Custom) UserData(_ *v1alpha5.KubeletConfiguration, _ []v1.Taint, _ map[s
 	}
 }
 
-func (c Custom) SSMAlias(_ string, _ *cloudprovider.InstanceType) string {
-	return "/unknown"
+func (c Custom) DefaultAMIs(_ string) []SSMAliasOutput {
+	return []SSMAliasOutput{{Name: "", Query: "/unknown", Requirements: scheduling.NewRequirements()}}
```

Comment:
Do you need to return something here?

Reply:
For custom AMIfamily we expect to return a value